### DISCUSSION TITLE
high memory utilization in C2

### DISCUSSION THREAD

**Discussion by sunqian2007** [filecoin-project/lotus#8112] [Storage Provider]
### Lotus Version

```text
1.10.0
```

### Describe the Bug

Out of memory: Killed process 1455117 (lotus-worker) total-vm:463450356kB, anon-rss:259575424kB, file-rss:0kB, shmem-rss:0kB, UID:0 pgtables:526152kB oom_score_adj:0

### Logging Information

```text
To Reproduce
After completing several seals and then getting ready for C2 phase, I have got the errors that my memory is out of use.
My machine is 256 GB ram, and the sector-size I set is 32GB
```

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
2 C2 badly timed will fail with 256gb ram

**Comment by sunqian2007** [comment]
any solutions?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
swap

**Comment by sunqian2007** [comment]
How much memory should it take?

**Comment by de-authority** [comment]
I don't think it's a bug. c2 will take more than 190GB. maybe 230+GB memory. see this:https://github.com/filecoin-project/community-china/blob/master/documents/resource/resource_usage/resource_usage.md

**Reply by Reiers** [reply]
Correct - I turned it into a discussion.

**Comment by LexLuthr** [comment]
@sunqian2007 Do you still need help with this problem?

**Comment by LexLuthr** [comment]
I am archiving this discussion as there has been no new update from the author for more than 7 days. Please open a new discussion if this is still reproducible on the latest stable release.
